translations/pt-br/02-o-que-é-open-source.md

```diff
@@ -0,0 +1,73 @@
+﻿# O que é Open Source?
+
+Neste capítulo, vamos mergulhar no conceito de open source, sua definição e sua história. Obter uma compreensão profunda do open source ajudará você a apreciar o contexto em que estará a colaborar e contribuindo.
+
+## Definição
+
+O termo "open source" refere-se a um tipo de software cujo código-fonte é disponibilizado ao público, permitindo que qualquer pessoa visualize, use, modifique e distribua o código. Isso é em contraste com o software proprietário, onde o código-fonte é mantido em segredo e apenas o programa compilado é distribuído. O princípio básico por trás do código aberto é a ideia de colaboração e a crença de que o compartilhamento de conhecimento e recursos leva a um software melhor, mais confiável e mais inovador.
+
+Um software de código aberto é construído com base na premissa de que, quando muitas pessoas trabalham juntas, elas podem criar algo maior do que a soma das suas partes. Os contribuidores para projetos de código aberto vêm de origens diversas e trazem uma ampla gama de habilidades, experiências e perspetivas para a mesa. Essa diversidade de pensamento e experiência leva a um software mais robusto, inovador e confiável.
```

Comment:
```suggestion
Um software de código aberto é construído com base na premissa de que, quando muitas pessoas trabalham juntas, elas podem criar algo maior do que a soma das suas partes. Os contribuidores para projetos de código aberto vêm de origens diversas e trazem uma ampla gama de habilidades, experiências e perspectivas para a mesa. Essa diversidade de pensamento e experiência leva a um software mais robusto, inovador e confiável.
```
I guess both are acceptable but "perspectivas" is more common. Also, for consistency sake, let's use "perspectivas" everywhere